tekton/pipeline_run_test.go

```diff
@@ -216,7 +219,24 @@ var _ = Describe("PipelineRun", func() {
 
 		It("can add the EC task bundle parameter to the PipelineRun", func() {
```

Comment:
The wording should be modified here. bundle -> git resolver information

Reply:
fixed